packages/devtools/lib/src/debugger/debugger.css

```diff
@@ -42,6 +48,29 @@
 
 .debugger-items-list {
     overflow-y: scroll;
+    cursor: pointer;
+}
+
+.open-popup {
+    z-index: 1000;
+    position: absolute;
+    top: 0px;
+    left: 5px;
+    height: 200px;
+    background-color: white;
+    list-style: none;
+    border: 1px solid #d1d5da;
```

Comment:
use an existing border color css variable if possible.

Reply:
This is the same border that's used on .menu

    border: 1px solid #d1d5da;

Reply:
Maybe we want to change .menu too?

Reply:
Created a popup-border-color that takes care of this.